internal/runner/session.go

```diff
@@ -41,6 +41,9 @@ func NewSession(envs []string, proj *project.Project, logger *zap.Logger) (*Sess
 func NewSessionWithStore(envs []string, proj *project.Project, owlStore bool, logger *zap.Logger) (*Session, error) {
 	sessionEnvs := []string(envs)
 
+	id := ulid.GenerateID()
+	sessionEnvs = append(sessionEnvs, fmt.Sprintf("RUNME_SESSION=%s", id))
```

Comment:
I added the RUNME_SESSION variable to infer its value later.

![image](https://github.com/stateful/runme/assets/43882/df006eba-d976-4a96-b0c3-658c2b0e53ab)